easybuild/easyconfigs/v/Visit/Visit-3.2.2-CrayGNU-21.09.eb

```diff
@@ -0,0 +1,84 @@
+# contributed by Jean Favre
+#
+easyblock = 'CMakeMake'
+
+name = 'Visit'
+version = '3.2.2'
+
+homepage = 'https://visit-dav.github.io/visit-website/'
+description = """VisIt is an Open Source, interactive, scalable, visualization,
+animation and analysis tool."""
+
+toolchain = {'name': 'CrayGNU', 'version': '21.09'}
+toolchainopts = {'usempi': True, 'verbose': False}
+
+#
+# Visit downloads all its dependencies by itself during the configuration.
+# The downloads may fail due to different reasons. So, we have pre-packaged
+# visit with the downloaded dependencies.
+#
+# Below you will find the steps to create the SOURCELOWER_TAR_GZ file used in
+# this recipe.
+#
+# mkdir /run/user/1100/VisIt; cd /run/user/1100/VisIt
+# wget  https://github.com/visit-dav/visit/releases/download/v3.2.2/visit3.2.2.tar.gz
+# tar xf visit3.2.2.tar.gz visit3.2.2
+# cd visit3.2.2
+# ./src/tools/dev/scripts/build_visit --console --no-visit --parallel \
+# --llvm --mesagl --vtk --makeflags -j32 --system-python --alt-python-dir /usr \
+# --openssl --qwt --qt --silo --hdf5 --cmake --icet --ospray --ispc --embree --tbb \
+# --xdmf --download-only
+# cd ..
+
+# tar cfz  visit-3.2.2.tar.gz visit3.2.2
+# mkdir -p $EASYBUILD_SOURCEPATH/v/Visit
+# cp visit-3.2.2.tar.gz $EASYBUILD_SOURCEPATH/v/Visit/
+#
+# NOTES:
+# * The new visit package is named visit-3.2.2.tar.gz, not visit3.2.2.tar.gz
+# as the original file.
+# * The download-only option must not be included in the actual build.
+# * The configuration options are the same as the ones below.
+# * We are using the system-python because one cannot compile python with xalt
+# enabled.
+# * In the preconfigopts commands below, there is an additional <<< "yes".
+# This is used to accept the Qt License. This is not needed when using
+# --download-only
+# * The original intent was to compile only the server-side components, disabling the
+# use of the GUI client for loggin nodes. Yet, this also removes the cli, preventing
+# us to run python-based batch-mode scripts. Thus, we build all components [default]
+
+
+# In order to use the pre-downloaded version use the source below
+sources = ['/apps/common/UES/easybuild/sources/%(nameletterlower)s/%(name)s/%(namelower)s-%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
```

Comment:
Thanks @jfavre: do you need to keep using CMake 3.14.5 to buil VisIt, or can you also use a more recent version?

Reply:
Visit uses its own global builder to compile all its dependencies and it actually requires cmake 3.14.7 which is very close to our own 3.14.5. I was able to compile with our own module to reduce the number of dependencies by one. I have not tested other CMake because the compilation is very long (about 1 hour)